M2/Macaulay2/d/actors3.d

```diff
@@ -1895,6 +1896,22 @@ map(e:Expr):Expr := (
      else WrongNumArgs(2,3));
 setupfun("apply",map);
 
+-- # typical value: applyPairs, BasicList, Function, List
+-- # typical value: applyPairs, Dictionary, Function, List
+-- # typical value: applyPairs, Thing, Function, Iterator
```

Comment:
I think it would be better to list these closer to where they are handled (e.g. near `mappairs` or `map`. We don't currently do this, but I'd like to have `locate` also show the location of compiled functions in the interpreter (the location is stored in `tvalues.m2` as  a comment right now).